tools/greptimedbcluster/clean-cluster/clean-cluster.sh

```diff
@@ -0,0 +1,149 @@
+#!/usr/bin/env bash
+
+set -e
+
+# Color definitions
+RED='\033[1;31m'
+GREEN='\033[1;32m'
+YELLOW='\033[1;33m'
+BLUE='\033[1;34m'
+MAGENTA='\033[1;35m'
+CYAN='\033[1;36m'
+RESET='\033[0m'
+
+# Input the GreptimeDBCluster name and namespace
+read -p "$(echo -e "${GREEN}Enter the name of the GreptimeDBCluster: ${RESET}")" NAME
+read -p "$(echo -e "${GREEN}Enter the namespace of the GreptimeDBCluster: ${RESET}")" NAMESPACE
+
+# Input the ETCD name and namespace
+read -p "$(echo -e "${GREEN}Enter the name of the ETCD StatefulSet: ${RESET}")" ETCD_NAME
+read -p "$(echo -e "${GREEN}Enter the namespace of the ETCD StatefulSet: ${RESET}")" ETCD_NAMESPACE
+
```

Comment:
Let's add a strong confirmation before actually run the logic. For example, ask user to type the namespace again